react/components/OrderInfo/index.tsx

```diff
@@ -38,7 +38,7 @@ const OrderInfo: FunctionComponent<Props> = ({
 
   return (
     <section>
-      <div className="mv6 w-80-ns w-90 center">
+      <div>
```

Comment:
Do you really need this div?

Reply:
Since I moved the css, I don't need this div. Can I remove it ?

Reply:
You tell me...